deepchecks/tabular/checks/train_test_validation/train_test_feature_drift.py

```diff
@@ -254,26 +255,39 @@ def condition(result: Dict) -> ConditionResult:
                     [col_name for col_name, fi in sorted(result.items(), key=lambda item: item[1]['Drift score'],
                                                          reverse=True)]
             columns_to_consider = columns_to_consider[:number_of_top_features_to_consider]
-            not_passing_categorical_columns = {column: f'{d["Drift score"]:.2}' for column, d in result.items() if
-                                               d['Drift score'] > max_allowed_categorical_score and
-                                               d['Method'] in SUPPORTED_CATEGORICAL_METHODS
-                                               and column in columns_to_consider}
-            not_passing_numeric_columns = {column: f'{d["Drift score"]:.2}' for column, d in result.items() if
-                                           d['Drift score'] > max_allowed_numeric_score
-                                           and d['Method'] in SUPPORTED_NUMERIC_METHODS
-                                           and column in columns_to_consider}
-            return_str = ''
-            if not_passing_categorical_columns:
-                return_str += f'Found categorical columns with {cat_method} above threshold: ' \
-                              f'{not_passing_categorical_columns}\n'
-            if not_passing_numeric_columns:
-                return_str += f'Found numeric columns with {num_method} above threshold: ' \
-                              f'{not_passing_numeric_columns}'
-
-            if return_str:
-                return ConditionResult(ConditionCategory.FAIL, return_str)
+            cat_drift_columns = {column: d['Drift score'] for column, d in result.items()
+                                 if d['Method'] in SUPPORTED_CATEGORICAL_METHODS}
```

Comment:
I understand this but maybe it would be more robust to pass a list of roles in the result...